foxtrot-core/src/main/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchQueryStore.java

```diff
@@ -110,8 +111,8 @@ public void save(String table, Document document) throws FoxtrotException {
                     .setType(ElasticsearchUtils.DOCUMENT_TYPE_NAME)
                     .setId(translatedDocument.getId())
                     .setTimestamp(Long.toString(timestamp))
-                    .setSource(convert(translatedDocument))
-                    .setConsistencyLevel(WriteConsistencyLevel.QUORUM)
```

Comment:
Why is this getting removed ?

Reply:
Doc for reference: https://www.elastic.co/guide/en/elasticsearch/reference/5.5/breaking_50_java_api_changes.html